what_url/ada.h

```diff
@@ -0,0 +1,6559 @@
+/* auto-generated on 2023-05-16 13:48:47 -0400. Do not edit! */
```

Comment:
Can you add a github workflow like the one we have in Rust?

https://github.com/ada-url/rust/blob/main/.github/update-ada.sh

Reply:
Thanks, I added one.